src/utils/feature-flag-utils.ts

```diff
@@ -41,44 +40,43 @@ export async function InitLdClient (): Promise<void> {
       streaming: true
     }
 
-    ldClient = initialize(envKey, user, options)
+    ldClient = initialize(ldClientId, context, options)
 
     try {
       await ldClient.waitForInitialization()
     } catch {
-      // shut down client -- `variation()` will return undefined values
+      // shut down client -- `variation()` will return undefined values (see GetFeatureFlag)
       await ldClient.close()
       // NB: LD logs its own errors
     }
   }
 }
 
 /**
- * An async method that updates the Launch Darkly user properties.
- * @param key a unique string identifying a user
- * @param email the user's email address
- * @param firstName the user's first name
- * @param lastName the user's last name
- * @param custom optional object of additional attributes associated with the user
+ * An async method that updates the Launch Darkly user and org properties.
+ * @param user an optional user context object
+ * @param org an optional organization context object
  */
-export async function UpdateLdUser (
-  key: string, email: string, firstName: string, lastName: string, custom: any = null
-): Promise<void> {
+export async function UpdateLdUser (user?: LDContext, org?: LDContext): Promise<void> {
   if (ldClient) {
-    const user: LDUser = { key, email, firstName, lastName, custom }
     try {
-      await ldClient.identify(user)
+      if (user && org) await ldClient.identify({ kind: 'multi', user, org })
+      else if (user) await ldClient.identify(user)
+      else if (org) await ldClient.identify(org)
     } catch {
-      // NB: LD logs its own errors
+      // do nothing -- LD logs its own errors
     }
```

Comment:
Sample multi context (generated by THIS code):

<img width="477" height="571" alt="image" src="https://github.com/user-attachments/assets/f4d1237b-e110-4e6e-9223-b74e67e73c3b" />